crates/chia-puzzles/src/derive_synthetic.rs

```diff
@@ -1,9 +1,8 @@
+use crate::standard::DEFAULT_HIDDEN_PUZZLE_HASH;
```

Comment:
Did you mean to reorder this?

I usually put crate dependencies below the rest (above is fine too, but at the very least separated via whitespace is nice).

Reply:
my ide does things